tools/mocha/src/lib/eventually.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const moment = require('abacus-moment');
+const edebug = require('abacus-debug')('e-abacus-mocha-eventually');
+const { defaults } = require('underscore');
+
+let _defaultPollingIntervalInMillis = 100;
+let _defaultTimeoutInMillis = 60000;
```

Comment:
We are not sure about this approach because it is not clear that eventually reads both environment variables and we are not able to see eventually configuration, that we are using, from the test file.